tests/integ/test_multithreading.py

```diff
@@ -664,6 +664,89 @@ def change_config_value(session_):
     )
 
 
+@pytest.mark.xfail(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="large query breakdown is not supported in local testing mode",
+    run=False,
+)
+def test_large_query_breakdown_with_cte(threadsafe_session):
```

Comment:
Can you a comment in the body of the test pointing to what causes large query breakdown to be triggered?

Reply:
@sfc-gh-aalam can you take care of this comment also